attic/rfc_metrics.md

```diff
@@ -67,10 +123,14 @@ mlr3measures
 Those packages either depend on or import the following other (additional) packages:
```

Comment:
The "Prior Art" Section is missing the following 
- Link to the wiki section of all fairness metrics
- A short explanation of how the API for the other Fairness Packages looks like. You can just copy over the relevant lines of R / Python / Julia Code, but it is useful for comparison